### PR TITLE
feat: NGS-bits SampleAncestry

### PR DESCRIPTION
**Pull request by tdayris**
This PR adds a second tool from [ngs-bits](https://github.com/imgag/ngs-bits/blob/master/doc/tools/SampleAncestry/index.md).

### QC

* [X] I confirm that I have followed the [documentation for contributing to `snakemake-wrappers`](https://snakemake-wrappers.readthedocs.io/en/stable/contributing.html).

While the contributions guidelines are more extensive, please particularly ensure that:
* [X] `test.py` was updated to call any added or updated example rules in a `Snakefile`
* [X] `input:` and `output:` file paths in the rules can be chosen arbitrarily
* [X] wherever possible, command line arguments are inferred and set automatically (e.g. based on file extensions in `input:` or `output:`)
* [X] temporary files are either written to a unique hidden folder in the working directory, or (better) stored where the Python function `tempfile.gettempdir()` points to 
* [X] the `meta.yaml` contains a link to the documentation of the respective tool or command under `url:`
* [X] conda environments use a minimal amount of channels and packages, in recommended ordering

## Summary by CodeRabbit

- **New Features**
	- Introduced `environment.linux-64.pin.txt` and `environment.yaml` files for easy environment setup on Linux.
	- Added a new tool, `NGS-bits SampleAncestry`, to estimate sample ancestry based on genetic variants.
	- Implemented a Snakemake wrapper for executing the `SampleAncestry` command.
	- Created a sample VCF file for testing the ancestry estimation functionality.

- **Bug Fixes**
	- Enhanced test coverage with new test functions for `SampleAncestry` and `autoindex` workflows.

- **Documentation**
	- Added comprehensive metadata for the `NGS-bits SampleAncestry` tool, detailing its functionality and requirements.

### CODE REVIEW

File: bio/ngsbits/sampleancestry/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    stderr=True,
+)
+extra = snakemake.params.get("extra", "")
+
+shell(
+    "SampleAncestry {extra} "
+    "-in {snakemake.input} "
+    "-out '{snakemake.output}' "
+    "{log}"
+)
```
[the remark this answers is not in the document]
Comment (tdayris) [reply bio/ngsbits/sampleancestry/wrapper.py:23]: Let Snakemake validate IO paths

File: bio/ngsbits/sampleancestry/test/Snakefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    input:
+        # Either a single VCF or a list of VCF files
+        "sample.vcf",
+    output:
+        "ancestry.tsv",
+    threads: 1
+    log:
+        "ancestry.log",
+    params:
+        extra="-min_snps 4 -build hg19",
```
[the remark this answers is not in the document]
Comment (tdayris) [reply bio/ngsbits/sampleancestry/test/Snakefile:11]: Let user decide which assembly or parameters they want. Supported genomes builds are given through tool's `url`.

File: bio/ngsbits/sampleancestry/test/Snakefile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+rule test_ngsbits_sampleancestry:
+    input:
+        # Either a single VCF or a list of VCF files
+        "sample.vcf",
```
[the remark this answers is not in the document]
Comment (tdayris) [reply bio/ngsbits/sampleancestry/test/Snakefile:4]: Declined.

File: bio/ngsbits/sampleancestry/wrapper.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tdayris) [reply bio/ngsbits/sampleancestry/wrapper.py:23]: Snakefile, vcf sample and tests are already available.

File: bio/ngsbits/sampleancestry/wrapper.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    stdout=False,
+    stderr=True,
+)
+extra = snakemake.params.get("extra", "")
+
+shell(
+    "SampleAncestry {extra} "
+    "-in {snakemake.input} "
+    "-out '{snakemake.output}' "
+    "{log}"
```
Comment (fgvieira) [inline bio/ngsbits/sampleancestry/wrapper.py:22]:
```suggestion
    "SampleAncestry {extra}"
    " -in {snakemake.input}"
    " -out {snakemake.output:q}"
    " {log}"
```

**Review by fgvieira — APPROVED** [review]

**Pull request merged**